.travis.yml

```diff
@@ -85,4 +91,8 @@ before_script:
       echo "require('awful'); tags = {}; tags[1] = awful.tag({'name'})" > ~/.config/awesome/rc.lua;
       awesome &
     fi
+  - if [[ -z "${CAPYBARA_CHROME_REMOTE}" ]]; then
```

Comment:
This check is the wrong direction - it's running docker when not needed, and not when it is needed